TrenDAP/Controllers/TrenDAPDBController.cs

```diff
@@ -133,14 +142,43 @@ public virtual ActionResult Get(int dataSourceID, string table = "")
                     return StatusCode(StatusCodes.Status500InternalServerError, ex);
                 }
             }
-
         }
 
-        private ActionResult GetOpenXDA(DataSource dataSource, string table) {
+        [HttpPost, Route("Channel/GetTrendChannels/{dataSourceID:int}")]
+        public virtual ActionResult Post(int dataSourceID, [FromBody] JObject filter)
```

Comment:
```suggestion
        public virtual ActionResult GetChannel(int dataSourceID, [FromBody] JObject filter)
```
`POST` is not a great name since that doesn't tell as what this does.